app/src/main/java/io/intrepid/pickpocket/codebreaker/CodeBreakerActivity.java

```diff
@@ -35,109 +43,168 @@
     TextView guessDigitFour;
     @BindView(R.id.lock_icon)
     CheckedTextView lockIcon;
+    @BindView(R.id.drawer_layout)
+    DrawerLayout drawerLayout;
+    @BindView(R.id.left_drawer)
+    LinearLayout linearLayout;
+
+
     private CodeBreakerContract.Presenter presenter;
+    private ViewAnimator viewAnimator;
 
-    public static Intent makeIntent(Context context) {
+    public static Intent makeIntent(Context context)
+    {
         return new Intent(context, CodeBreakerActivity.class);
     }
 
     @Override
-    public void onCreate(Bundle savedInstanceState) {
+    public void onCreate(Bundle savedInstanceState)
+    {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_code_breaker);
         ButterKnife.bind(this);
 
         presenter = new CodeBreakerPresenter();
         presenter.setView(this);
+        drawerLayout.setScrimColor(Color.BLACK);
+        lockIcon.setOnClickListener(new View.OnClickListener()
+        {
+            @Override
+            public void onClick(View v)
+            {
+                drawerLayout.openDrawer(Gravity.LEFT);
+                viewAnimator.showMenuContent();
+            }
+        });
+
+        viewAnimator = new ViewAnimator(this, presenter.getCodeBreakerGuessList(), drawerLayout, this);
```

Comment:
Kind of confused by this, it looks like we're not updating the guess list. Also it would be better to have the presenter call a function like: 

setupViewAnimator(List<GuessList> guessList){
     viewAnimator = new ViewAnimator(this, guessList, drawerLayout, this);
}

Where possible avoid having the activity ASK for data and instead have the Presenter TELL the view what to do.